source/elements/oneDNN/source/api/primitives/rnn.rst

```diff
@@ -0,0 +1,16 @@
+..
+  Copyright 2019 Intel Corporation
+
+.. default-domain:: cpp
+
+RNN
+~~~
+
+.. doxygenclass:: dnnl::rnn_flags
+   :project: oneDNN
+   :members:
+
+.. doxygenclass:: dnnl::rnn_direction
+   :project: oneDNN
+   :members:
+
```

Comment:
The rendering is also broken:

![image](https://user-images.githubusercontent.com/15225631/75934224-c4b72700-5e30-11ea-992a-2edb69c68cbb.png)

And it also is missing most of the [RNN classes](https://intel.github.io/mkl-dnn/group__dnnl__api__rnn.html).

Reply:
Warnings resolved by Roma (doxygenenum directive instead of doxygenclass). I don't see any missing classes for RNN, because the `dnnl_api_rnn` group ends on line 550. There are only the two enum classes within that group.

Reply:
Filled in the rest of the RNN and Attribute APIs. Please review.